modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-move-status.adoc

```diff
@@ -1,63 +1,64 @@

-this command assumes the "kafka" namespace, but you can use a "namespace/" to

-specify internal namespaces.

-

-    rpk cluster partitions move-status

-    rpk cluster partitions move-status foo bar kafka_internal/tx

-

-The "--partition / -p" flag can be used with topics to additional filter

-requested partitions:

-

-    rpk cluster partitions move-status foo bar --partition 0,1,2

-

-The output contains the following columns, where PARTITION-SIZE is in bytes.

-Using -H, it prints the partition size in a human-readable format

-

-    NAMESPACE-TOPIC

-    PARTITION

-    MOVING-FROM

-    MOVING-TO

-    COMPLETION-%

-    PARTITION-SIZE

-    BYTES-MOVED

-    BYTES-REMAINING

-

-Using "--print-all / -a" the command additionally prints "RECONCILIATION STATUSES"

-which reveals internal states on how the ongoing reconciliations work for debugging

-purposes. That is, reported errors don't necessarily mean real problems.
-
-== Usage
-
-[,bash]
-----
-rpk cluster partitions move-status [flags]
-----
-
-== Flags
-
-[cols="1m,1a,2a]
-|===
-|*Value* |*Type* |*Description*
-
-|`-h, --help` |- |Help for move-status.
-
-|`-H, --human-readable` |- |Print the partition size in a human-readable form.
-
-|`-p, --partition` |strings |Partitions to print the ongoing movements (repeatable).
-
-|`-a, --print-all` |- |Print internal states about movements for debugging.
-
-|`--config` |string |Redpanda or rpk config file; default search paths are ~/.config/rpk/rpk.yaml, $PWD, and /etc/redpanda/`redpanda.yaml`.
-
-|`-X, --config-opt` |stringArray |Override rpk configuration settings; '-X help' for detail or '-X list' for terser detail.
-
-|`--profile` |string |rpk profile to use.
-
-|`-v, --verbose` |- |Enable verbose logging.
+= rpk cluster partitions move-status
+:description: rpk cluster partitions move-status
+
+Show ongoing partition movements.
+
+By default this command lists all ongoing partition movements in the cluster. Topics can be specified to print the move status of specific topics. By default, this command assumes the `kafka` namespace, but you can use a `namespace/` to specify internal namespaces.
+
+```bash
+    rpk cluster partitions move-status
+```
+
+```bash
+rpk cluster partitions move-status foo bar kafka_internal/tx
+```
+
+The `--partition / -p` flag can be used with topics to additional filter requested partitions:
+
+
+```bash
+rpk cluster partitions move-status foo bar --partition 0,1,2
+```
+
+The output contains the following columns with PARTITION-SIZE in bytes. Using -H, it prints the partition size in a human-readable format
+
+- Namespace-Topic
+- Partition
+- Moving-From
+- Moving-To
+- Completion-%
+- Partition-Size
+- Bytes-Moved
+- Bytes-Remaining
+
+Using `--print-all / -a` the command additionally prints the column `RECONCILIATION STATUSES`, which reveals the internal status of the ongoing reconciliations. Reported errors do not necessarily mean real problems.
```

Comment:
```suggestion
Using `--print-all / -a` the command additionally prints the column `Reconciliation Statuses`, which reveals the internal status of the ongoing reconciliations. Reported errors do not necessarily mean real problems.
```

Reply:
For better readability, it's a best practice not to use all caps in docs even if the UI or code displays it that way.

Reply:
Yup. The CLI description almost always use all caps for the columns. I had to remove them manually. This one was prob missed